config/common.go

```diff
@@ -77,7 +77,7 @@ func (cs *commonStore) set(newCfg *model.Config, allowEnvironmentOverrides bool,
 		}
 	}
 
-	if err := persist(cs.RemoveEnvironmentOverrides(newCfg)); err != nil {
+	if err := persist(cs.RemoveNonPersistable(newCfg)); err != nil {
```

Comment:
I seem to recall that we did want to persist the current configuration -- effectively using the config as a cache for the event that the feature flag system went down and so to avoid "flopping" if a pod restarts. Can you remind me what's actually going to happen?

Reply:
There will be some persistence in order to make sure the cluster is in sync. Not part of this PR, will be part of the management system PR. It might involve changing this code so that it is persisted in some cases (such as only on MM cloud where we have DB control)